examples/OSMPDummySensor/OSMPDummySensor.cpp

```diff
@@ -256,6 +269,27 @@ void rotatePoint(double x, double y, double z,double yaw,double pitch,double rol
 
 }
 
+/*!
+     * \brief 2D rotation matrix function. Calculates local X,Y coordinates and relative yaw angle from reference.  
+     * \param x Global relative x-coordinate.

```

Comment:
What do you mean by "Global relative coordinates"? Global coordinates should be fixed

Reply:
Its just relative coordinates unrotated. Just wanted to be extra clear that its the coordinates before the rotation has been done.